prover/run.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```

Comment:
Can you soft remove this file and also remove the changes made to zkevm.bin

Reply:
Done :)